README.md

```diff
@@ -352,7 +356,7 @@ Decoder checks for all required well-formedness errors, including all "subkinds"
 
 After well-formedness is verified, basic validity errors are handled as follows:
 
-* Invalid UTF-8 string: Decoder always checks and returns invalid UTF-8 string error.
+* Invalid UTF-8 string: Decoder checks and returns invalid UTF-8 string error by default.
```

Comment:
Sounds a bit awkward because "by default" was added to the end.

Reply:
Good catch!